tests/libres_tests/res/enkf/test_res_config.py

```diff
@@ -505,7 +505,6 @@ def test_res_config_dict_constructor(setup_case):
 
     # add missing entries to config file
     with open(config_file_name, "a+") as ert_file:
-        ert_file.write("JOB_SCRIPT ../../../script.sh\n")
```

Comment:
I believe what is going on here is that your refactoring of the test left cwd in a different place once you gave it to `ResConfig` so that `../../../script.sh` is no longer pointing at the scrip.

Reply:
hmm it doesn't seem like the refactoring changed the path for the chdir call to me, but of course i may be wrong. i'm gonna investigate!